vault/policy_store.go

```diff
@@ -22,9 +22,16 @@ const (
 	// policyCacheSize is the number of policies that are kept cached
 	policyCacheSize = 1024
 
+	// defaultPolicyName is the name of the default policy
+	defaultPolicyName = "default"
+
 	// responseWrappingPolicyName is the name of the fixed policy
 	responseWrappingPolicyName = "response-wrapping"
 
+	// controlGroupPolicyName is the name of the fixed policy for control group
+	// tokens
+	controlGroupPolicyName = "control-group"
```

Comment:
I think we need to add this to the list of non-settable policies, otherwise tokens with this policy name will be able to get around decrementing use counts

Reply:
Ah yeah, I moved that over later instead of in the normal chunk...will do.